src/progress/web/method/cgidefs.i

```diff
@@ -0,0 +1,175 @@
+/*********************************************************************
+* Copyright (C) 2000 by Progress Software Corporation. All rights    *
+* reserved. Prior versions of this work may contain portions         *
+* contributed by participants of Possenet.                           *
+*                                                                    *
+*********************************************************************/
+/*-------------------------------------------------------------------------
+
+File: src/web/method/cgidefs.i
+
+Description: Global include file for all WEB and CGI variables.
+
+Notes:
+  The NEW option must be specified when this file is included by the
+  initial calling procedure.   
+  
+  Associated procedures for manipulating these variables are
+  found in cgiutils.i
+
+  This file includes src/web/method/proto.i to define function prototypes.
+
+Author: B.Burton
+
+Created: 05/14/96
+
+---------------------------------------------------------------------------*/
+
+/* Stream where we write all web output */
+&IF DEFINED(WEBSTREAM) = 0 &THEN
+  &GLOBAL-DEFINE WEBSTREAM STREAM WebStream
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) From src/progress/pct/pctWSComp.p - Preprocessor variable webstream should be SCOPED-DEFINED [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=rssw-oe-main%3Aeu.rssw.antlr.proparse.checks.GlobalDefine)